QualityControl/test/setup/testServerSetup.js

```diff
@@ -62,9 +68,10 @@ export async function setupServerForIntegrationTests() {
     console.error('        ', pageerror);
   });
   page.on('console', (msg) => {
-    for (let i = 0; i < msg.args().length; ++i) {
-      console.log(`        ${msg.args()[i]}`);
-    }
+    const filterRegex = /JSHandle:render: \d\.\d+ ms/;
+    const lines = msg.args().filter((arg)=> !filterRegex.test(arg.toString()));
```

Comment:
If `msg.args()` could potentially be undefined or not an array, adding a check would prevent errors
```suggestion
    const args = msg.args() || [];
    const lines = args.filter((arg)=> !filterRegex.test(arg.toString()));
```